src/app/components/person/person.tsx

```diff
@@ -0,0 +1,49 @@
+import styles from './person.module.css';
+
+interface PersonProps {
+  fullName: string;
+  twitterUrl: string;
+  linkedinUrl: string;
+  imageUrl: string;
+}
+
+export default function Person({
+  fullName,
+  twitterUrl,
+  linkedinUrl,
+  imageUrl,
```

Comment:
Image url is unused. Did you mean to use this in "src" below?

Reply:
Yeah, should be fixed now!